### ISSUE TITLE
[Enhancement] Implement IExtendedEntityProperties (Console spam if mod removed)

### ISSUE THREAD

**Issue by KryptonCaptain** [satyrnidae/wolfarmor#8]
Removed mod from a 1.7 test instance, and found the console spamming this when I loaded back in. 
```
[Client thread/WARN]: Skipping Entity with id wolfarmor.Wolf
[Server thread/WARN]: Skipping Entity with id wolfarmor.Wolf
```
Anything you can do about that?

**Comment by CenturionFox** [comment]
Hrm.

This is due to my replacement of existing wolves with the wolfarmor variant.  For version 1.10.2+, gsbaoge's fix (Issue #4) should remedy this as I'll replace the custom wolf entities with the minecraft.wolf entity, just with an extra capability.

The problem is that the 1.7 version of forge does not support capabilities; those were added in 1.8.  What I can do, however, is add a "removal" mode to the mod which would replace all the wolfarmor wolves with normal wolves, and probably empty their inventories as well.  The user would have to enable the removal mode and run minecraft once with the mod still enabled, but after that the mod could just be uninstalled as normal.

Plus that fixes the unfortunate side effect of those wolf entities just being removed from the world.  Probably should have thought of that initially.

Thanks for reporting this!

**Comment by satyrnidae** [comment]
Closing this because I'm sunsetting 1.7.10
